packages/web/babel.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Babel preset to provide TypeScript support and module/nomodule for Nx.
+ */
+
+module.exports = function (api: any, options: {}) {
```

Comment:
This is our own preset, so we can easily modify/improve our default setup without needing to worry about migrations.